Hm, this causes another permission error on my machine:

```
python3 ./src/trello2md.py --output build/export.md export.json --comments --header
Sucessfully translated to "build/export.md"!
docker run --env "UNAME=1000:1000" --volume /home/philipp/git/trello2md:/data marcelhuberfoo/pandoc-gitit pandoc -f markdown -t latex build/export.md -o build/export.pdf --template=tex/trello.latex --latex-engine=xelatex "--variable=title:export" 
pandoc: build/export.pdf: openBinaryFile: permission denied (Permission denied)
Makefile:77: recipe for target 'build/export.pdf' failed
make: *** [build/export.pdf] Error 1
```

However, I have found it to work with changing the command to:

```
MDPROC = docker run --volume $(shell pwd):/data:z marcelhuberfoo/pandoc-gitit pandoc
```

(see [here](https://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/)). Does that work for you, too?

--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ $(error Please install $(MDPROC) e.g. sudo apt install pandoc)
 endif
 
 else
-MDPROC = docker run --env "UNAME=$(shell id -u):$(shell id -g)" --volume $(shell pwd):/data marcelhuberfoo/pandoc-gitit pandoc
+MDPROC = docker run --env "UNAME=$(shell id -u):$(shell id -g)" --volume $(shell pwd):/data:z marcelhuberfoo/pandoc-gitit pandoc
 MDPARAMS_PDF = -f markdown -t latex $(BUILD_DIR)/$(notdir $<) -o $@ --template=$(PANDOCTEMPLATE_TEX) --latex-engine=xelatex "--variable=title:$(basename $(notdir $@))" $(MDPARAMS_PDF_ADDITIONAL)
 MDPARAMS_HTML = -f markdown -t html5 $(BUILD_DIR)/$(notdir $<) -o $@
 MDPARAMS_TEX = -f markdown -t latex $(BUILD_DIR)/$(notdir $<) -o $@

